### ISSUE TITLE
Is this project going to be deprecated/archived anytime soon?

### ISSUE THREAD

**Issue by vladimiry** [ProtonMail/proton-contacts#394]
I mean the same what happened with https://github.com/ProtonMail/proton-mail-settings.

If not, is the respective link getting back to the apps dropdown list?

Related commit: https://github.com/ProtonMail/react-components/commit/af3f0a191f6777563c9a41300214c38389f59e94

**Comment by EpokK** [comment]
Yes @vladimiry we plan to close this repository since all contacts feature have been integrated inside ProtonMail, ProtonCalendar and ProtonDrive.
